src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -330,6 +326,11 @@ public void setField(String name, String value) {
         Objects.requireNonNull(name, "field name must not be null");
         Objects.requireNonNull(value, "field value must not be null");
 
+        if(value.isEmpty()) {
```

Comment:
Sure? - Wie wanted to force the caller to use clearField and not this setField method. Therefore Objects.requireNonNull